app/components/shared/work_history_and_unpaid_experience_item_component.html.erb

```diff
@@ -6,7 +6,7 @@
     <h3 class="govuk-heading-s govuk-!-margin-bottom-0">
       <%= title %>
 
-      <% if @editable %>
+      <% if @editable && edit_path %>
```

Comment:
Can you add one component spec for this?

Reply:
I don't think it's worth adding one but if you think it's necessary, I'll sort it out.